File: spec/python/test_user_type.py
import unittest

from user_type import UserType

class TestUserType(unittest.TestCase):
    def test_user_type(self):
        r = UserType.from_file("src/repeat_until_s4.bin")

        self.assertEqual(r.one.width, 0x42)
        self.assertEqual(r.one.height, 0x1337)

File: spec/lua/test_user_type.lua
local luaunit = require("luaunit")

require("user_type")

TestUserType = {}

function TestUserType:test_user_type()
    local r = UserType:from_file("src/repeat_until_s4.bin")

    luaunit.assertEquals(r.one.width, 0x42)
    luaunit.assertEquals(r.one.height, 0x1337)
end
